instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/SnsAdviceBridge.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.awssdk.v2_2;
+
+public final class SnsAdviceBridge {
```

Comment:
Sorry, I totally missed this in your last PR, when you introduced `SqsAdviceBridge`: these bridge classes are only here to make javaagent & muzzle work correctly, they should not be a part of the library instrumentation API. WDYT about moving the class (without changing the package name) to the javaagent module?

Reply:
I think that would be a fine idea. Would look a bit odd, but reducing the public API surface is always good.

Reply:
Done in b1267c06be0b4187b2015f4f8e080e047431f976, just moving the files w/o any other changes seems to have worked without any problems.

Reply:
Thanks!